service/controller.go

```diff
@@ -552,13 +576,27 @@ func (s *service) CreateVolume(
 		if contentSource != nil {
 			volumeSource := contentSource.GetVolume()
 			if volumeSource != nil {
-				Log.Printf("volume %s specified as volume content source", volumeSource.VolumeId)
-				return s.Clone(req, volumeSource, name, size, storagePool)
+				cloneResponse, err := s.Clone(req, volumeSource, name, size, storagePool)
+				if err != nil {
+					return nil, err
+				}
+
+				cloneResponse.Volume.AccessibleTopology = volumeTopology
+
+				return cloneResponse, nil
 			}
 			snapshotSource := contentSource.GetSnapshot()
 			if snapshotSource != nil {
 				Log.Printf("snapshot %s specified as volume content source", snapshotSource.SnapshotId)
-				return s.createVolumeFromSnapshot(req, snapshotSource, name, size, storagePool)
+				snapshotVolumeResponse, err := s.createVolumeFromSnapshot(req, snapshotSource, name, size, storagePool)
+				if err != nil {
+					return nil, err
+				}
+
+				snapshotVolumeResponse.Volume.AccessibleTopology = volumeTopology
+
+				Log.Printf("[FERNANDO] Snapshot Volume Response: %+v", snapshotVolumeResponse)
```

Comment:
`"FERNANDO"` 😂

Reply:
He had to leave his footprint, but a bit too obvious :)

Reply:
Removed 😞